docker/docker-compose-build.yml

```diff
@@ -14,6 +14,12 @@ services:
       context: ../
       dockerfile: docker/images/python/Dockerfile.prod
 
+  python-stage:
+    image: fidals/stb:stage
```

Comment:
Why do we need separate image for stage? The stage and prod have the same image

Reply:
@artemiy312 , stage is created to expose given branch different from master.
`fidals/stb:prod` image contains master branch.
`fidals/stb:stage` contains a given one.

I'll provide comment about it